src/Symfony/Bundle/SecurityBundle/DependencyInjection/Security/Factory/SignatureAlgorithmFactory.php

```diff
@@ -19,6 +19,8 @@
 /**
  * Creates a signature algorithm for {@see OidcTokenHandler}.
  *
+ * @internal
+ *
  * @experimental
```

Comment:
can be removed I suppose

Reply:
still, let's keep it to spot experimental things quickly